components/panel/src/index.tsx

```diff
@@ -31,7 +31,7 @@ const StyledPanel = styled('div')(
     },
 
     color: WHITE,
-    background: TURQUOISE,
+    background: BUTTON_COLOUR,
```

Comment:
I would prefer we either hard code this to '#00703c' here or do an update to govuk-colours to export a semantic name that matches whatever is in govuk-frontend.

Probably the former as I have limited time and no support to maintain this at the moment

Reply:
@penx updated to hex as requested, thanks for the prompt response